core-rust/state-manager/src/query/transaction_identifiers.rs

```diff
@@ -1,13 +1,15 @@
 use enum_dispatch::enum_dispatch;
 
-use crate::{AccumulatorState, CommittedTransactionIdentifiers};
+use crate::{CommittedTransactionIdentifiers, LedgerHashes};
 
 #[enum_dispatch]
 pub trait TransactionIdentifierLoader {
-    fn get_top_transaction_identifiers(&self) -> Option<CommittedTransactionIdentifiers>;
-    fn get_top_accumulator_state(&self) -> AccumulatorState {
+    // TODO(no-accu-hash): it's worth having `pub struct StateVersion(u64);` at this point
```

Comment:
_(note from author)_

I want to do this in a post-feature-refactor pretty much right after we merge this to develop.